auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -359,7 +363,9 @@ public Void then(@NonNull Task<GoogleApiClient> task) throws Exception {
      * silently.
      *
      * @param activity the calling {@link Activity}.
+     * @deprecated use {@link #delete(FragmentActivity)} instead
      */
+    @Deprecated
     public Task<Void> delete(@NonNull Activity activity) {
```

Comment:
@samtstern SGTM, I'll leave things deprecated so we know to remove that stuff if we go that route, but I keep things clean in the meantime.